### PR TITLE
Add Stream51.

### PR DESCRIPTION
**Pull request by arthurdouillard**
- [X] Datasets
- [X] Bounding boxes support
- [x] In-code doc
- [x] Web doc

### CODE REVIEW

**Comment by arthurdouillard** [comment]
Related to #67

File: docs/_tutorials/datasets/dataset.rst [hunk 1]
```diff
@@ -36,6 +36,8 @@ Existing Datasets
 +----------------------+------------+------------+--------------------+--------+
 | **CORe50-v2-391**    | 50         | 224x224x3  | YES                | Images |
 +----------------------+------------+------------+--------------------+--------+
+| **Stream-51**        | 51         | ?x?x3      | YES                | Images |
```
Comment (TLESORT) [inline docs/_tutorials/datasets/dataset.rst:39]: Est ce qu'il faut pas en plus faire une partie dans scenarios suite? ya plusieur scenarios possible dans Stream-51 non?

Comment (arthurdouillard) [reply docs/_tutorials/datasets/dataset.rst:39]: Yes I though about putting in another PR

Comment (arthurdouillard) [reply docs/_tutorials/datasets/dataset.rst:39]: See #100 , if you want to can do it :p

**Comment by arthurdouillard** [comment]
I'll need to check one thing, don"t merge

**Comment by arthurdouillard** [comment]
Doing full download in tests of large dataset is honestly a bit annoying. It's going to take age to end. We may want to do it, but it'll be in a latter PR #101

**Pull request merged**